Dockerfile.alpine

```
@@ -0,0 +1,42 @@
+FROM python:3.7.3-alpine3.9 AS build
+
+ARG DOCKER_COMPOSE_GITSHA=unknown
+ENV DOCKER_COMPOSE_GITSHA=$DOCKER_COMPOSE_GITSHA
+
+ENV BUILD_BOOTLOADER=1
+
+RUN apk add --no-cache \
+    bash \
+    build-base \
+    ca-certificates \
+    curl \
+    gcc \
+    git \
+    libc-dev \
+    libffi-dev \
+    libgcc \
+    make \
+    musl-dev \
+    openssl \
+    openssl-dev \
+    python2 \
+    python2-dev \
+    zlib-dev && \
+    pip install virtualenv==16.2.0 tox==2.1.1
```

I'd keep the `pip install` separate; it likely won't overwrite files that were installed by `apk add`, so doing it in a separate step will add a new layer, but won't make the image bigger.

Also worth to keep the same convention as for `apk add`, and split the packages that will be installed to one-per-line, sorted alphabetically (it makes the Dockerfile longer, but can help making it more maintainable)

```suggestion
RUN pip install \
    tox==2.1.1 \
    virtualenv==16.2.0
```

If these packages are expected to be updated individually, could even be two `RUN` lines

---

Thanks for that one!